examples/vm-cirros-arm64.yaml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
what is arm64 specific about this example?

Reply:
> what is arm64 specific about this example?

There are 4 main different between this yaml and vm-cirros.yaml
- It use UEFI boot (Default boot method is BIOS). ARM64 only support vm UEFI boot.
- The cpu model is host-passthrough (Default cpu model is host-model). "host-model" is not support well on ARM64 platform
- the request memory has increased to 256M, as minimal memory size for UEFI boot with edk2 package should be larger than 128M.
- Disable autoattachGraphicsDevice, as vga device is not support the build-in qemu-kvm.

Reply:
> I have a try on this branch,however,get this error:{"component":"virt-launcher","kind":"","level":"error","msg":"Failed to sync vmi","name":"vmi9","namespace":"default","pos":"server.go:161","reason":"virError(Code=67, Domain=10, Message='unsupported configuration: CPU mode 'host-passthrough' for aarch64 qemu domain on aarch64 host is not supported by hypervisor')","timestamp":"2020-09-01T03:41:54.921063Z","uid":"3f60e204-ec04-11ea-8a09-52540031d801"}.
> Do you know why?

I did not meet this error before. Can you boot VM via libvirt (not use kubevirt) on the bare metal server?

Reply:
> There are 4 main different between this yaml and vm-cirros.yaml
>
> It use UEFI boot (Default boot method is BIOS). ARM64 only support vm UEFI boot.
> The cpu model is host-passthrough (Default cpu model is host-model). "host-model" is not support well on ARM64 platform
> the request memory has increased to 256M, as minimal memory size for UEFI boot with edk2 package should be larger than 128M.
> Disable autoattachGraphicsDevice, as vga device is not supported by the build-in qemu-kvm on ARM.

@zhlhahaha out of this list, I'm only seeing the 256M memory increase. Shouldn't we need to specify something like ```spec.firmware.bootloader.efi:{}``` here to ensure uefi is used over bios? same with host-passthrough and autoattachgraphicsDevice